spec.html

```diff
@@ -21281,6 +21297,14 @@ <h1>Static Semantics: AllPrivateIdentifiersValid</h1>
         1. Append to _newNames_ the elements of PrivateBoundIdentifiers of |ClassBody|.
         1. Return AllPrivateIdentifiersValid of |ClassElementList| with argument _newNames_.
       </emu-alg>
+
+      <emu-grammar>RelationalExpression : PrivateIdentifier `in` ShiftExpression</emu-grammar>
+      <emu-alg>
+        1. If StringValue of |PrivateIdentifier| is in _names_, return *true*.
+        1. Return *false*.
+      </emu-alg>
+
+      <p>For all other grammatical productions, recurse on subexpressions/substatements, passing in the _names_ of the caller. If all pieces return *true*, then return *true*. If any returns *false*, return *false*.</p>
```

Comment:
I think this was accidentally left here while rebasing, but it has been replaced by the first catch-all algorithm of https://tc39.es/ecma262/#sec-static-semantics-allprivateidentifiersvalid.